src/components/Newsletter/Newsletter.style.tsx

```diff
@@ -30,14 +34,25 @@ export const StyledNewsletter = styled.div`
       line-height: 2rem;
       height: 3.25rem;
       outline: 0;
-      border: 1px solid rgba(0, 0, 0, 0.12);
+      border: 1px solid ${COLOR_MAP.GRAY.BASE};
+      border-right: none;
       border-top-left-radius: 4px;
       border-bottom-left-radius: 4px;
       appearance: none;
       font-size: 0.875rem;
       box-sizing: border-box;
       font-family: 'Roboto', 'Helvetica', 'Arial', sans-serif;
 
+      ::placeholder {
+        color: #828282;
+        font-size: 15px;
+      }
+
+      &:hover {
+        border: 1px solid black;
+        border-right: none;
+      }
```

Comment:
oh I see that it was on purpose, never mind

Reply:
yeah, this black hover is the default state on the material ui autocomplete component and i spent way tooooo much time trying to figure out how to get rid of it. eventually threw in the towel and just made the email input match